coralproject/talk
# [CORL-2729] Moderation Queue Focus

## What does this PR do?
This PR fixes a bug in which upon navigating to `/admin/moderate`, focus immediately shifts to the first moderation card, which is an anti-pattern for accessibility. 

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Ensure that there are multiple comments in the moderation queue
2. As an admin or moderator, navigate to `/admin/moderate`
3. Press the tab key
4. Observe that the focus begins at the first element on the page (the nav at the top) and cycles through the different nav options, rather than skipping them and jumping to the beginning of the mod queue
5. Observe that using the "j" and "k" keys still cycles through mod cards
 
 
## How do we deploy this PR?
No special considerations should be necessary
